airflow/providers/microsoft/azure/example_dags/example_local_to_adls.py

```diff
@@ -18,11 +18,19 @@
 import os
 
 from airflow import models
+from airflow.operators.python import PythonOperator
+from airflow.providers.microsoft.azure.hooks.azure_data_lake import AzureDataLakeHook
 from airflow.providers.microsoft.azure.transfers.local_to_adls import LocalToAzureDataLakeStorageOperator
 from airflow.utils.dates import days_ago
 
 LOCAL_FILE_PATH = os.environ.get("LOCAL_FILE_PATH", 'localfile.txt')
-REMOTE_FILE_PATH = os.environ.get("REMOTE_LOCAL_PATH", 'remote')
+REMOTE_FILE_PATH = os.environ.get("REMOTE_LOCAL_PATH", 'remote.txt')
+
+
+def remove_file():
```

Comment:
What do you think to add the operator that will do this action?  I think it would be good if the example DAGs did not include tricks with calling the Hook directly, but if possible they would use operators.

Reply:
Ok. I will add it